javascript/sdk/lib/types.ts

```diff
@@ -56,6 +56,12 @@ export interface Account {
   company: string;
 }
 
+export interface VerifiedUser {
+  account_id: string;
+  token: string;
+  user: User;
```

Comment:
It's the flat properties of the user, not the user object itself

Reply:
resolved